content/definition.it.md

```diff
@@ -1,14 +1,44 @@
 +++
-title = "La definizione di Sorgente Etica"
+title = "La definizione di Sorgente Etica (SE)"
 template = "definition"
+heading="La definizione di Sorgente Etica"
 +++
 
-Un Software è considerato Sorgente Etica (_Ethical Source_) se rispetta i seguenti criteri:
+Un Software è considerato **Open-Source Etico** (EOS) se rispetta *tutti* i seguenti criteri:
```

Comment:
I would not translate this at all and call it `Ethical Open-Source` instead.

Reply:
I think that is pretty subjective, we can leave it in English or do a full translation, which is not bad at all

Reply:
I do not think it is. Search engines do not understand translation well, and figuring out the english name to google something up is complicated.

Overall, I try to consider names of projects/groups/initiatives as they were people's name. As an example, we do not translate Matthew in Matteo (or viceversa)

Reply:
We don't maybe, but usually in other languages (for instance, German), _everything_ gets translated, even names. An example? GDPR becomes DSGVO (Datenschutzgrundverodnung)